bluemira/equilibria/optimisation/harmonics/harmonics_approx_functions.py

```diff
@@ -680,3 +677,5 @@ def plot_psi_comparision(
     plot4 = plt.subplot2grid((5, 4), (3, 2), rowspan=2, colspan=1)
     plot4.set_title("SH Approximation, Vacuum Psi")
     plot4.contour(grid.x, grid.z, vac_psi_app, levels=clevels, cmap=cmap, zorder=8)
+
+    plt.show()
```

Comment:
Could we have the plt.show as optional? Eg put show=True in the function definition. Secondly could we return the axes so that the plots can be modified if desired by the user before showing